gobblin-api/src/main/java/gobblin/annotation/Stable.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Denotes an API that is still stable and should not change.
+ */
+@Documented @Retention(value=RetentionPolicy.CLASS) @Target(value=ElementType.TYPE)
+public @interface Stable {
```

Comment:
Yes, I think we need Stable because none of our APIs are currently annotated and still undergo changes. I think marking something as Stable is a deliberate decision which we have to make on a case-by-case basis.